core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -195,9 +208,20 @@ Map<String,String> modifyProperties(Consumer<Map<String,String>> mapMutator)
    *        {@code <ip address>:<port>}
    * @return A list of active scans on tablet server.
    */
+  @Deprecated
   List<ActiveScan> getActiveScans(String tserver)
       throws AccumuloException, AccumuloSecurityException;
 
+  /**
+   * List the active scans on a tablet server.
+   *
+   * @param server server type and address
+   * @return A list of active scans on tablet server.
+   * @since 3.0.0
+   */
+  List<ActiveScan> getActiveScans(Server server)
```

Comment:
Unless there is a specific reason to use list, using a more generic type gives more flexibility w/ changing the impl

```suggestion
  Collection<ActiveScan> getActiveScans(Server server)
```

Reply:
Implemented in ea67851.